### ISSUE TITLE
[BUG] Localized name not displaying correctly

### ISSUE THREAD

**Issue by asherber** [finale-lua/lua-scripts#692]
For the scripts that newly take advantage of localization options, returning `t.menu, t.menu, t.desc` doesn't seem to have the desired effect. In my Lua menu, I'm seeing the name of the script file (e.g., `baseline_move_reset.lua`) instead of the expected text. 

Win 11, Fin 27.4.1, RGPLua 0.70. For some reason, I seem to be unable to take a screenshot inside of Finale at the moment. I'm using finale_lua_menus.txt to organize things under Plug-ins | Lua, but I see the same behavior when I delete that file.

cc: @rpatters1

**Comment by rpatters1** [comment]
Do any of the additional menu items appear?

**Comment by rpatters1** [comment]
OK, I have reproduced it. Will explore later today.

**Comment by rpatters1** [comment]
This issue is that RGP Lua pre-0.71 is search for this line of code:

```lua
function plugindef()
```

I have fixed this issue by modifying the localized `plugindef` function definitions as follows:

```lua
function plugindef(locale)
--[[
-- This comment allows RGP Lua pre-0.71 to find the plugindef function
function plugindef()
--]]
```

As you can see, this allows any version of RGP Lua to find a valid compilable Lua function. (It also works on JW Lua, which appears only to search for `function plugindef`, like 0.71 does now.)

The only reason I am not sure it will work is I need to be sure the bundler does no strip out these comments.

**Comment by rpatters1** [comment]
Maybe we could skip the stripping of comments for the script itself, and only do it for libraries. Or maybe we could actually inject this comment into each plugindef function.

**Comment by asherber** [comment]
I think it's likely that the bundler will strip that comment, but I'm not at my computer. And injecting it would solve the issue for dist scripts but not src.

**Comment by asherber** [comment]
The bundler does strip that comment, but it's easy to tell it not to strip comments from `plugindef()`. Is that the approach you'd like to take?

In terms of aesthetics, would JW/RGPLua be happy with just a single-line comment, or does it need to be formatted just that way? Does `function` need to be flush left?

**Comment by rpatters1** [comment]
It has to be a multiline comment, but the function statement could be preceded by whitespace, e.g.

```lua
--[[
   function plugindef()
--]]
```

I think it looks better with no whitespace, but ymmv.

The reason it has to be a multiline comment is that the parser looking for the function is quite basic. It strips anything after `--`, but it doesn't know about multiline comments. So it takes the above `function plugindef()` as the beginning of the function. As luck would have it, `--]]` is interpreted by Lua either as the terminator of a multiline comment or as a single line comment.

**Comment by rpatters1** [comment]
I think not stripping comments out of `plugindef` makes sense.

**Comment by rpatters1** [comment]
Fixed in #694
